solution-base/deps.txt

```diff
@@ -1,10 +1,6 @@
-kubedb/operator:v0.13.0-rc.0
-kubedb/redis:5.0.3-v1
-kubedb/redis_exporter:v0.21.1
-kubedb/mongo:4.0-v1
-kubedb/mongo-tools:4.0-v1
-kubedb/busybox:latest
-kubedb/mongodb_exporter:v1.0.0
 pravega/zookeeper-operator:0.2.7
 banzaicloud/kafka-operator:v0.11.2
 gcr.io/kubebuilder/kube-rbac-proxy:v0.4.0
+bitnami/mongodb:4.0.14-debian-9-r24
+bitnami/minideb:stretch
```

Comment:
there isn't a versioned alternative provided. what can be done is to remove create a fixed vesion to be referenced in  'registry.scality.com`

Reply:
Please add a followup ticket for this

Reply:
created ticket: ZENKO-3614

Reply:
Another approach you could use is to rely on the SHA256, instead of a version tag (which is not a reproducible entity either, FWIW).

It can be easily retrieved with:

```bash
MINIDEB_DIGEST=$(skopeo inspect docker://bitnami/minideb:stretch | jq -r '.Digest')
docker pull "bitnami/minideb@$MINIDEB_DIGEST"
```